test.js

```diff
@@ -3,3 +3,9 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var fooStag = '' ;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.staging.codacy.org/gh/machadoit-test-org/eslint-test-stag/pullRequest?prid=13589)
```suggestion
var fooStag = "" ;
```